packages/scandipwa/src/component/ResetAttributes/ResetAttributes.component.js

```diff
@@ -48,7 +49,13 @@ export class ResetAttributes extends PureComponent {
                 </div>
                 <div block="ResetAttributes" elem="AttributeText">
                     <span block="ResetAttributes" elem="AttributeLabel">{ `${attribute_label}: ` }</span>
-                    <span block="ResetAttributes" elem="AttributeOption">{ `${selectedOption.label}` }</span>
+                    <span block="ResetAttributes" elem="AttributeOption">
+                        { typeof selectedOption.label === 'string' ? (
+                            <Html content={ selectedOption.label } />
+                        ) : (
+                            selectedOption.label
+                        ) }
```

Comment:
In order to improve readability and extensibility, please move this to a separate method and use conditional blocks instead of the inline ternary operator.